examples/gallery-dev.html

```diff
@@ -1,21 +1,27 @@
 <html>
 
 <head>
-  <title>Esri Vector Tiles</title>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>Esri Leaflet Vector Basemap Gallery</title>
 
   <!-- Load Leaflet from CDN -->
-  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" />
-  <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.0/dist/leaflet.css" />
+  <script src="https://unpkg.com/leaflet@1.5.0/dist/leaflet.js"></script>
```

Comment:
Should these be v1.7.1?

Reply:
Good point, I wasn't sure how we'd want to deal with these. Should they approximate what is in package.json? Should we just point to local node_modules installations instead of CDN?